### PR TITLE
VAGOV-6329: Add 8 other VISN 4 VAMC system menus.

### PR DESCRIPTION
**Pull request by kevwalsh**
## Description
We're adding 8 more VAMC systems, each with their own menu. Eventually this will need to be refactored to scale for ~144 systems. 

https://va-gov.atlassian.net/browse/VAGOV-6329 

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by jhonny218 — APPROVED** [review]

File: src/site/stages/build/drupal/graphql/navigation-fragments/facilitySidebar.nav.graphql.js [hunk 1]
```diff
@@ -9,6 +9,14 @@ const { camelize } = require('./../../../../../utilities/stringHelpers');
 const FACILITY_MENU_NAMES = [
   'pittsburgh-health-care',
   'va-altoona-health-care',
+  'va-butler-health-care',
+  'va-coatesville-health-care',
+  'va-altoona-health-care',
```
Comment (swirtSJW) [inline src/site/stages/build/drupal/graphql/navigation-fragments/facilitySidebar.nav.graphql.js]: 'va-altoona-health-care' is in there twice.

**Review by swirtSJW — CHANGES_REQUESTED** [review]
One duplicate entry.   And if possible it would be good to have them alphasorted so the list as it grows  is easier to manage.

File: src/site/stages/build/drupal/graphql/navigation-fragments/facilitySidebar.nav.graphql.js (hunk 1 above)
Comment (kevwalsh) [reply src/site/stages/build/drupal/graphql/navigation-fragments/facilitySidebar.nav.graphql.js]: it's alpha sorted now that the dupe is removed.

**Review by swirtSJW — APPROVED** [review]
Looks good

**Pull request merged**